internal/wclayer/cim/process.go

```diff
@@ -34,10 +33,6 @@ func processBaseLayerHives(layerPath string) ([]pendingCimOp, error) {
 	}
 
 	hivesDirInfo := &winio.FileBasicInfo{
-		CreationTime:   windows.NsecToFiletime(time.Now().UnixNano()),
```

Comment:
I believe it will be `January 1, 1601` as mentioned [here](https://learn.microsoft.com/en-us/windows/win32/api/minwinbase/ns-minwinbase-filetime?redirectedfrom=MSDN).